src/main/webapp/resources/js/updateEmployee.js

```diff
@@ -1,17 +1,18 @@
 /**
- * Created by leo on 31/05/15.
+ * @author leo
+ * @date 31/05/15 21:57
  */
 function handleClick() {
-    $(this).siblings().on("click", handleClick);
-    $(this).off("click");
-    //console.log($(this).attr("value"));
-    var empId = $(this).closest("tr").find("td:nth-child(1)").text();
-    var firstName = $(this).closest("tr").find("td:nth-child(2)").text();
-    var lastName = $(this).closest("tr").find("td:nth-child(3)").text();
-    //console.log("Update " + empId + " to :" + firstName + ", " + lastName);
-    $.post("UpdateEmployee", {"empID": empId, "firstName": firstName, "lastName": lastName},
-        function () {
-        }, "json");
+	$(this).siblings().on("click", handleClick);
+	$(this).off("click");
+	//console.log($(this).attr("value"));
+	var empId = $(this).closest("tr").find("td:nth-child(1)").text();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/lcappuccio/crud-application/file/2269579977/issues/source?bid=3039358&fileBranchId=3184972#l9)